grammar/hooks.js

```diff
@@ -152,7 +152,8 @@ class FormulaParser {
                     value: res.val,
                     isArray: res.isArray,
                     isRangeRef: !!FormulaHelpers.isRangeRef(arg),
-                    isCellRef: !!FormulaHelpers.isCellRef(arg)
+                    isCellRef: !!FormulaHelpers.isCellRef(arg),
+                    position: FormulaHelpers.retrievePosition(arg)
```

Comment:
Instead of calling it `position`, I will rename it to `ref` (reference position)